doc/sources/installation/installation-windows.rst

```diff
@@ -140,146 +153,150 @@ For installation instructions, see :ref:`install-win-dist`. See also :ref:`dev-i
 
 .. _kivy-dependencies:
 
-Kivy's dependencies
--------------------
+Installing Kivy's dependencies
+--------------------------
 
-We offer wheels for Kivy and its dependencies, separately, so only desired
-dependencies need be installed. The dependencies are offered as
-optional sub-packages of kivy_deps, e.g. ``kivy_deps.sdl2``.
+We offer the above wheels for the Kivy core and for its below dependencies separately, so that you can install only the desired
+dependencies. The dependencies are offered as sub-packages, starting with ``kivy_deps``, for example ``kivy_deps.sdl2``.
 
 .. note::
 
-    In Kivy 1.11.0 we transitioned the kivy Windows dependencies from the
-    `kivy.deps.xxx` namespace stored under `kivy/deps/xxx` to the
-    `kivy_deps.xxx` namespace stored under `kivy_deps/xxx`.
+    In Kivy 1.11.0 we replaced the dot in ``kivy.deps`` with an underscore. So, instead of ``kivy.deps.xxx``, stored under ``kivy/deps/xxx`` it is now ``kivy_deps.xxx``, stored under ``kivy_deps/xxx``.
     See `here <https://github.com/kivy/kivy/wiki/Moving-kivy.garden.xxx-to-kivy_garden.xxx-and-kivy.deps.xxx-to-kivy_deps.xxx#kivy-deps>`_
     for more details.
 
-On Windows, we provide the following dependency wheels:
+So, the following are the dependency wheels which we provide for Windows:
+
+* `gstreamer <https://gstreamer.freedesktop.org>`_ (optional)
+
+  `gstreamer` is an optional dependency which is only needed for audio/video support.
+  It can be installed with  ``python -m pip install kivy_deps.sdl2``
 
-* `gstreamer <https://gstreamer.freedesktop.org>`_ for audio and video.
+* `ffpyplayer <https://pypi.org/project/ffpyplayer/>`_ (optional)
+
+  `ffpyplayer` is an alternative optional dependency for audio or video.
+  It can be installed with ``python -m pip install ffpyplayer``.
 
-  ``gstreamer`` is an optional dependency which is only needed for audio/video support.
-  ``ffpyplayer`` is an alternate dependency for audio or video. It can be installed with
-  ``pip install ffpyplayer``.
 * `glew <http://glew.sourceforge.net/>`_ and/or
-  `angle <https://github.com/Microsoft/angle>`_ for OpenGL.
+  `angle <https://github.com/Microsoft/angle>`_
+  
+  These are for `OpenGL <https://en.wikipedia.org/wiki/OpenGL>`_. They can be installed with ``python -m pip install kivy_deps.glew`` and/or ``python -m pip install kivy_deps.angle``.
 
   One can select which of these to use for OpenGL using the
-  ``KIVY_GL_BACKEND`` environment variable by setting it to ``glew``
-  (the default), ``angle``, or ``sdl2``. ``angle`` is a substitute for ``glew``.
-* `sdl2 <https://libsdl.org>`_ for control and/or OpenGL.
+  ``KIVY_GL_BACKEND`` environment variable: By setting it to ``glew``
+  (the default), ``angle``, or ``sdl2``. Here, ``angle`` is a substitute for ``glew``.
```

Comment:
P.S. glew and angle may be installed at the same time without issues.

Reply:
The and/or already indicates that, but i added an extra verbal comment.

Thanks a lot for your detailed input!